extensions/adobe/experience/target/segmentevent.schema.json

```diff
@@ -0,0 +1,49 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/target/activity/activityevent/segmentevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Activity Segment Event",
+  "type": "object",
+  "description": "Represents Activity Segment Event.",
+  "definitions": {
+    "segmentevent": {
+      "properties": {
+        "xdm:segmentId": {
```

Comment:
ID and use the full term "identifier" in title and description.

Reply:
Please see the defined segment definitions in \context and consider how to use that.

Reply:
I've used "https://ns.adobe.com/xdm/context/segmentidentity"